paper/myst.yml

```diff
@@ -1,49 +1,55 @@
 # See docs at: https://mystmd.org/guide/frontmatter
 version: 1
 project:
-  id: ubcgif-2020-kang-etal-exploration-geophysics
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  id: ubcgif-2020-kang-etal-exploration
```

Comment:
hmm, it doesn't seem to like this id, but I am not too sure why. Any help from @rowanc1 or @stevejpurves would be appreciated!